<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_login">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Digite Seu Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_senha">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Digite sua senha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_entrar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
